core/src/main/java/io/questdb/TelemetryConfigLogger.java

```diff
@@ -120,13 +164,21 @@ private TableWriter updateTelemetryConfig(
                 final boolean enabled = telemetryConfiguration.getEnabled();
                 if (cursor.hasNext()) {
                     final Record record = cursor.getRecord();
-                    final boolean _enabled = record.getBool(1);
                     final Long256 l256 = record.getLong256A(0);
+                    final boolean _enabled = record.getBool(1);
                     final CharSequence _questDBVersion = record.getSymA(2);
+                    final Utf8Sequence _instanceName = record.getVarcharA(5);
+                    final CharSequence _instanceType = record.getSymA(6);
+                    final Utf8Sequence _instanceDesc = record.getVarcharA(7);
 
-                    // if the configuration changed to enable or disable telemetry
+                    // if the configuration or instance information changed (enable or disable telemetry, for example),
                     // we need to update the table to reflect that
-                    if (enabled != _enabled || !Chars.equalsNc(questDBVersion, _questDBVersion)) {
+                    if (enabled != _enabled
```

Comment:
what is this going to do if telemetry config have more than one row already?

Reply:
it is enough to take only the last row.
this listener callback runs after every update (+ on startup too).

Reply:
yeah, sorry, i missed the "limit" clause